apps/web/src/features/spaces/components/SpaceAddressBook/EditContactDialog.tsx

```diff
@@ -122,7 +122,7 @@ const EditContactDialog = ({ entry, onClose }: EditContactDialogProps) => {
             <Typography mb={2}>Edit contact details. Anyone in the space can see it.</Typography>
             <Stack spacing={3}>
               <Box pt={1}>
-                <AddressInputReadOnly address={entry.address} />
+                <AddressInputReadOnly address={entry.address} chainId={entry.chainIds[0]} />
```

Comment:
A bit hacky but we know every entry has at least one network and the name is the same regardless of chainId.